core-bundle/src/Resources/contao/library/Contao/System.php

```diff
@@ -160,6 +160,11 @@ protected function import($strClass, $strKey=null, $blnForce=false)
 		{
 			$container = static::getContainer();
 
+			if (null === $container)
+			{
+				throw new \RuntimeException('The Symfony container is not available, did you initialize the Contao framework?');
```

Comment:
Are you sure that this is no BC break?

Reply:
At least not for Contao 4. We'd always call `$container->has()` on a `null` object, and the exception message (`calling has() on null`) is very unintuitive.